account_bank_statement_import/models/res_partner_bank.py

```diff
@@ -33,6 +33,18 @@ class ResPartnerBank(models.Model):
     sanitized_acc_number = fields.Char(
         'Sanitized Account Number', size=64, readonly=True,
         compute='_get_sanitized_account_number', store=True, index=True)
+    enforce_unique_import_lines = fields.Boolean(
```

Comment:
Actually it is only (as yet) implemented for camt files, but there is no reason why it could not be extended to other formats.